home.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles/home.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <title>Mauritania palms</title>
+</head>
+<body>
+    <header class="header">
+        <img src="images/navBar.svg" alt="navBar" class="navBar">
+        <img src="images/closeIcon.svg" alt="closeIcon" class="closeIcon dis-none">
+        <div class="navUl dis-none" id="navUl">
+            <a href="./home.html" class="home upperCase">home</a>
+            <a href="./about.html" class="about upperCase">about</a>
+        </div>
+        <div class="desktopHeader">
+        <img src="./images/logo.jpg" alt="logo" class="logo">
+        <div class="desktopUl">
+            <a href="" class="desktopUl-li upperCase">about</a>
+            <a href="" class="desktopUl-li upperCase">sponsors</a>
+            <a href="" class="desktopUl-li upperCase">Persons</a>
+            <a href="" class="desktopUl-li upperCase">contact us</a>
+        </div>
+    </div>
+    </header>
+    <main id="main">
+        <section class="mainInfos">
+            <h1 class="title upperCase">"Hello,Date Lovers"</h1>
+            <h2 class="title-2 upperCase">Dates Festival 2021
+               Mauritania Tagant.</h2>
+               <p class="mainPara">
+                Dates are the fruit of the date palm tree, which is grown in tropical regions of the world.
+                Dates are healthy and delicious and have become very popular recently.
+               </p>
+               <div class="mainDate">
+                <h3 class="eventDate">2021.12.23(FRI) ~ 25(SUN)</h3>
+                <h4 class="eventPlace">@ National Dates Festival of Mauritania, Tagant.</h4>
+            </div>
+        </section>
```

Comment:
The **alignment** and **typography**  of your _hero_ / _introduction_ section do not match the provided [design template](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015). We actually want our design to as much as possible closely mimic the design of the provided template. Kindly ensure to improve upon these:
- [ ] Add appropriate **padding at the top** of the contents of this section, as seen in the original design
- [ ] The typography(**font sizes**, **font-weight**, **font color**) of the _small line of text_, the _page title_,  _block of text_, _medium line of text in bold letters_, and the _smaller line of text_, should closely match the provided template design.
- [ ] Contents of this section should be **evenly spaced** (paddings/ margins), as can be seen in the provided template design.

**Your hero design:**
![image](https://user-images.githubusercontent.com/12745474/133656702-a7df5966-fe46-4800-a1f4-765c346d37f5.png)


**The original hero design:**
![image](https://user-images.githubusercontent.com/12745474/132607606-41b0af31-fa67-4194-8d5d-5874348ab4d8.png)

Reply:
![di](https://user-images.githubusercontent.com/44745855/133761398-98e7d28e-6730-40b1-8229-a719350a574a.png)

As you can see all the `hx` here have the same margin compare with the original to improve it